packages/utils/src/evmChainIds.ts

```diff
@@ -0,0 +1,20 @@
+import type { EvmChainId } from '@shapeshiftoss/types'
+import { KnownChainIds } from '@shapeshiftoss/types'
+
+export const isEvmChainId = (
+  maybeEvmChainId: string | EvmChainId,
+): maybeEvmChainId is EvmChainId => {
+  return evmChainIds.includes(maybeEvmChainId as EvmChainId)
+}
+
+export const evmChainIds = [
+  KnownChainIds.EthereumMainnet,
+  KnownChainIds.AvalancheMainnet,
+  KnownChainIds.OptimismMainnet,
+  KnownChainIds.BnbSmartChainMainnet,
+  KnownChainIds.PolygonMainnet,
+  KnownChainIds.GnosisMainnet,
+  KnownChainIds.ArbitrumMainnet,
+  KnownChainIds.ArbitrumNovaMainnet,
+  KnownChainIds.BaseMainnet,
+] as const
```

Comment:
Yes, this is duplicated but fixes this guy when running `tsx ./scripts/generateAssetData/generateAssetData.ts ` following portals utils consolidation in web:

https://github.com/ethers-io/ethers.js/discussions/4163

We already have it fixed in web through react-app-rewired config but trying to get this to work in scripts is borderline impossible

Reply:
Actually moved it directly into the script in #7744 so as not to introduce 2 sources of truth to the rest of the project

Reply:
i.e can remove this after rebase

Reply:
Oh i see this PR is after that change. Ignore me.